### ISSUE TITLE
[Feature Request] 

### ISSUE THREAD

**Issue by jevylux** [joBr99/nspanel-lovelace-ui#703]
### FEATURE DESCRIPTION
In Iobroker allow to create a card that displays all data from an alias based on a socket

### ADDITIONAL CONTEXT
When I create an Device that has the role socket, i can define more than just the state.
I would like to have an entity that displays not only a button to switch the socket on and off, but also displays additional information ie, current power, total energy consumption, actual power etc

### PANEL / FIRMWARE VERION
EU Panel  with tasmota version 12.3.1
<img width="878" alt="Screenshot 2023-01-19 at 11 28 39" src="https://user-images.githubusercontent.com/198311/213418791-2378d6ec-97ae-47a8-824e-f6173c066d75.png">

**Comment by Armilar** [comment]
That's correct, the virtual device can hold more than just the .SET value. But you could also display the individual values in a cardGrid. However, apart from the switch, you cannot use the values of the device type socket. It would have to be separate info aliases.

Maybe @joBr99 has another idea? Possibly a popup for consumption properties?
